frame/contracts/src/storage.rs

```diff
@@ -225,4 +327,15 @@ where
 			.and_then(|i| i.as_alive().map(|i| i.code_hash))
 			.ok_or(ContractAbsentError)
 	}
+
+	/// Fill up the queue in order to exercise the limits during testing.
+	#[cfg(test)]
+	pub fn fill_queue_with_dummies() {
+		let queue: Vec<_> = (0..T::DeletionQueueDepth::get()).map(|_| DeletedContract {
+			pair_count: 0,
+			trie_id: vec![],
+		})
+		.collect();
+		DeletionQueue::put(queue);
+	}
 }
```

Comment:
should add a newline to the file ending. check ur editor to automate this for you